packages/trader/src/sass/app/_common/components/full-page-modal.scss

```diff
@@ -18,6 +18,7 @@
         position: absolute;
         left: calc(100vw - 70%);
         top: calc(100vh - 70%);
+        z-index: 1;
```

Comment:
@zaki-binary this has been moved to components, you can find it as `dialog.scss`. 
please do update it there.

Reply:
I can update it, but I'm not sure if the fix would work outside of trader context. We might have to refactor it completely.

Reply:
@zaki-binary what is an `inconsistent stylesheet`? yes please update it. cause this file that you changed has been removed.